src/org/labkey/test/params/experiment/SampleTypeDefinition.java

```diff
@@ -30,6 +30,9 @@
     // Indicates which parent aliases reference 'exp.dataInputs' instead of 'exp.materialInputs'
     private Set<String> _dataParentAliases = new HashSet<>();
 
+    private MetricUnit _inventoryMetricUnit;
+    private String _labelColor;
```

Comment:
Maybe add a note indicating that these are used only by Freezer Manager and Sample Manager

Reply:
Updated. I limited the comment to the SampleManager module because that is the only module that takes advantage of them.